src/main/scala/nl/knaw/dans/easy/fedoratobag/Configuration.scala

```diff
@@ -68,6 +69,7 @@ object Configuration {
       new URI(properties.getString("bag-index.url")),
       File(properties.getString("staging.dir")),
       AbrMappings(cfgPath / "EMD_acdm.xsl"),
+      properties.getString("export.states").replace(" ", "").split(",").toList,
```

Comment:
Nit picking: A slightly shorter variant:
```suggestion
      properties.getString("export.states").split(",").toList.map(_.trim),
```